FinSearchUnified/BusinessLogic/ExportService.php

```diff
@@ -0,0 +1,208 @@
+<?php
+
+namespace FinSearchUnified\BusinessLogic;
+
+use Doctrine\ORM\EntityNotFoundException;
+use Exception;
+use FINDOLOGIC\Export\Data\Item;
+use FINDOLOGIC\Export\Helpers\EmptyValueNotAllowedException;
+use Shopware\Models\Article\Article;
+use Shopware\Models\Category\Category;
+use Shopware\Models\Customer\Customer;
+
+class ExportService
+{
+    /**
+     * @var \Shopware\Models\Customer\Repository
+     */
+    protected $customerRepository;
+
+    /**
+     * @var \Shopware\Models\Article\Repository
+     */
+    protected $articleRepository;
+
+    /**
+     * @var FindologicArticleFactory
+     */
+    protected $findologicArticleFactory;
+
+    /**
+     * @var string
+     */
+    public $shopKey;
```

Comment:
In all projects that I am involved it is always `shopkey`. Also the parameter name is `shopkey`, so this property should also be named `shopkey` for consistency.